source/brailleDisplayDrivers/hims.py

```diff
@@ -272,12 +272,13 @@ def registerAutomaticDetection(cls, driverRegistrar: bdDetect.DriverRegistrar):
 
 	@classmethod
 	def getManualPorts(cls):
-		return braille.getSerialPorts(filterFunc=lambda info: "bluetoothName" in info)
+		return braille.getSerialPorts()
 
 	def __init__(self, port="auto"):
 		super(BrailleDisplayDriver, self).__init__()
 		self.numCells = 0
 		self._model = None
+		self._serData = b''

```

Comment:
We generally prefer full words for names just to reduce ambiguity (e.g. numCells is fine), however we user `serial` in full elsewhere

```suggestion
		self._serialData = b''
```